### PR TITLE
adding check disk space section to upgrade

### PR DESCRIPTION
**Pull request by pablodcar**

### CODE REVIEW

**Review by paigehargrave — APPROVED** [review]

File: modules/ROOT/pages/upgrade.adoc [hunk 1]
```diff
@@ -25,6 +25,22 @@ If you are upgrading from a prior version of Anypoint Platform PCE, you must fir
 
 . Perform a backup of your system as described in xref:backup-and-disaster-recovery.adoc[Configure Backup and Restore]. If you are upgrading from an older version that requires multiple intermediate upgrades, you must backup each intermediate version before proceeding to the next.
 
+. Verify that there is adequate disk space available for the upgrade, which is at least half the amount specified in xref:prereq-hardware.adoc[Hardware Prerequisites]. For example, for `/var/lib/gravity`, the recommended amount of disk space is 100Gb, so ensure that you have 50Gb of available space. Run the following command to check the amount of available space:
```
Comment (federicoboerr) [inline modules/ROOT/pages/upgrade.adoc:28]: @pablodcar are we sure about the `50Gb of available space`?
How much is it actually required? (at lease according to our automation)

Comment (pablodcar) [reply modules/ROOT/pages/upgrade.adoc:28]: Testing in a real cluster ...

File: modules/ROOT/pages/upgrade.adoc [hunk 2]
```diff
@@ -25,6 +25,22 @@ If you are upgrading from a prior version of Anypoint Platform PCE, you must fir
...
++
+----
+sudo df -h /var/lib/gravity/
```
Comment (federicoboerr) [inline modules/ROOT/pages/upgrade.adoc]: `sudo` not needed

File: modules/ROOT/pages/upgrade.adoc (hunk 1 above)
Comment (federicoboerr) [reply modules/ROOT/pages/upgrade.adoc:28]: the gc has to be ran after `Create a new folder and untar the new installer:`

File: modules/ROOT/pages/upgrade.adoc (hunk 2 above)
Comment (paigehargrave) [reply modules/ROOT/pages/upgrade.adoc]: Removed `sudo`.

**Comment by federicoboerr** [comment]
This is moving away from public docs

**Pull request closed without merging**